CMakeLists.txt

```diff
@@ -189,28 +192,67 @@ target_include_directories(onnx PUBLIC ${ONNX_ROOT} "${CMAKE_CURRENT_BINARY_DIR}
 target_link_libraries(onnx PUBLIC onnx_proto)
 
 if(BUILD_PYTHON)
```

Comment:
I prefer to no support building python extension without going through setup.py. All these PY_EXT_SUFFIX, PYTHON_EXECUTABLE cmake variables should be directly passed from setup.py (passing via cmake flags).